src/resolvers/projectResolver.test.ts

```diff
@@ -1050,7 +1047,3 @@ function editProjectTestCases() {
     );
   });
 }
-
-before(async () => {
-  apolloServer = await createServerWithDummyUser();
```

Comment:
JFYI
I noticed that createServerWithDummyUser() is useless in our tests so I deleted them.

Reply:
Ohhhhh. I really thought it was for axios to work. Got it!